tests/test_compare.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import os
+from pychimera import patch_environ, enable_chimera
+
+patch_environ()
+# enable_chimera()
+
+import chimera
+from compare_equal import compare
+
+
+def test_pdb():
+    """Test fot the function compare_pdb"""
+
+    assert compare.compare_pdb("tests/data/mol1", "tests/data/mol2")
+    assert not compare.compare_pdb("tests/data/mol1", "tests/data/mol3")
+    assert not compare.compare_pdb("tests/data/mol2", "tests/data/mol3")
+
+
+def test_mol():
+    """Test fot the function compare_mol"""
+
+    path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
It's common to have a small "data provider" function for tests that simply gets the full path to your test data. Something like:

```
def test_data(path):
    base = os.path.dirname(os.path.abspath(__file__))
    return os.path.join(base, path)
```

Reply:
Okay! Thank you, it seems useful. I have added a 'data provider' function in a new commit.